test/module/irohad/torii/processor/query_processor_test.cpp

```diff
@@ -170,7 +173,7 @@ TEST_F(QueryProcessorTest, GetBlocksQueryNoPerms) {
 
   EXPECT_CALL(*wsv_queries, getSignatories(kAccountId))
```

Comment:
This should not be called, same as in previous test. Please remove this expect, and `wsv_queries` field from fixture.